newIDE/app/src/InstructionOrExpression/EnumeratedInstructionOrExpressionMetadata.js

```diff
@@ -37,27 +43,20 @@ export type EnumeratedInstructionOrExpressionMetadata =
   | EnumeratedInstructionMetadata
   | EnumeratedExpressionMetadata;
 
-export const filterEnumeratedInstructionOrExpressionMetadataByScope = <
-  +T: EnumeratedInstructionOrExpressionMetadata
```

Comment:
I'm pretty sure this broke flow but CRA wouldn't compile this after I touched the file.

Reply:
What was the error? Can you try to put back the original typing and paste the error here?

Reply:
Huh the error seems to be gone now. Weird.